lightning/src/events/mod.rs

```diff
@@ -929,6 +929,10 @@ pub enum Event {
 		///
 		/// [`ChannelManager::send_payment`]: crate::ln::channelmanager::ChannelManager::send_payment
 		payment_hash: PaymentHash,
+		/// The total amount that was paid, across all paths.
```

Comment:
LGTM. Might be worth noting that fees are excluded from this number? We do that on the equivalent `Route` method.